Update: locales in spanish

DanielRondonGarcia opened the pull request:
Hello @Yidadaa ,

I have created a fork of the ChatGPT-Next-Web project and added or updated the locales to support the Spanish language. I have done some testing and everything seems to be working correctly.

I hope you find these updates useful, and if you have any questions or suggestions, please don't hesitate to let me know.

Thank you for your time and effort on this project!

Best regards

Yidadaa commented:
Hi @DanielRondonGarcia,

Thank you so much for contributing the Spanish translation! I really appreciate your efforts and time put into this project.

I will review your code as soon as possible and merge it into the project.

Yidadaa commented on app/locales/es.ts:
```diff
@@ -0,0 +1,154 @@
+      Close: "Cerrar",
+    },
+    Lang: {
+      Name: "Idioma",
```
Please set the name here to "Language". The reason is that users may accidentally set the language option to a completely unfamiliar language. Keeping this option as "Language" ensures that they can change it back in such a situation.

DanielRondonGarcia commented:
Hi @Yidadaa,

I have made the following updates in this pull request:

- Changed the 'Name' option in language files to consistently display 'Language', following your idea to make 'Language' universal, which I believe enhances the user experience.
- Fixed the locale issues in 'tw' and 'cn' files that were mistakenly changed.

> I really appreciate your efforts in improving the user experience, and it was a pleasure working on this enhancement. Please review these changes and let me know if there are any additional modifications needed. Looking forward to your feedback.
